meson.build

```diff
@@ -239,7 +239,13 @@ if cxx.has_header('sys/time.h')
 endif
 
 
-install_man('docs/ots-sanitize.1')
+install_man([
+  'docs/ots-idempotent.1',
+  'docs/ots-perf.1',
+  'docs/ots-sanitize.1',
+  'docs/ots-side-by-side.1',
+  'docs/ots-validator-checker.1',
+])
```

Comment:
Some of these tools are built conditionally, so man pages should only be installed when the tools are installed.

Reply:
Thanks; I‘ll look at the details and revise.